src/provider.coffee

```diff
@@ -5,6 +5,25 @@ extensions        = require './extensions'
 
 
 
+paramMatches = (param, duplicate) ->
+  if Array.isArray param
```

Comment:
query params (`duplicate`) is the one more likely to be an array, but checking both probably makes sense. in this case you seem to be bailing out if the body param is not an array, but the query param may still be

the `every` is requiring that every param is duplicated, also that the matches would be in the same order. I think you want `some`?


Anyway - I think this covers the cases
```coffee
paramIsDuplicated = (param, duplicate) ->
  param = Array.isArray(param) ? param : [param];
  duplicate = duplicate ? [];
  duplicate = Array.isArray(duplicate) ? duplicate : [duplicate];

  param.some (param) ->
    -1 != duplicate.indexOf param
```